<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UITaskDialogView</class>
 <widget class="QDialog" name="UITaskDialogView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Task</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_task_name">
      <property name="text">
       <string>Task Name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_task_name">
      <property name="placeholderText">
       <string>Enter task name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_macro">
      <property name="text">
       <string>Macro:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_macro">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_start_time">
      <property name="text">
       <string>Start Time:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEdit_start_time">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_end_time">
      <property name="text">
       <string>End Time:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEdit_end_time">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_interval">
      <property name="text">
       <string>Interval:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_interval">
      <item>
       <widget class="QSpinBox" name="spinBox_interval_value">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_interval_unit">
        <item>
         <property name="text">
          <string>second(s)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>minute(s)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>hour(s)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>day(s)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>week(s)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>month(s)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_frequency">
      <property name="text">
       <string>Frequency:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="spinBox_frequency">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_buttons">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ok">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>